app/models/statement_item.rb

```diff
@@ -1,3 +1,9 @@
+# StatementItem
+#
+# * Is the join table of a many-to-many relation between declarations and
+#   statements
+# * A declaration could have two statement line items max, one for the original
+#   payment and a potential second one if it’s clawed back
```

Comment:
We could perhaps represent the idea of a clawback better via a reference than a status. Having a self-referential foreign key (`clawed_back_from_id`) would give us more information than a raw `state`, because:

* it tells us which item we're clawing back
* it easily allows us to find all clawbacks (or items that have been clawed back, depending on the direction of the key)